### ISSUE TITLE
Handlebar execution on command line is broken.

### ISSUE THREAD

**Issue by PeterCamilleri** [PeterCamilleri/mysh#14]
Code now falls over, goes bang:
```
mysh>echo {{ 'foo' }}
C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:38:in `block in do_process_handlebars': undefined method `mysh_eval' for #<#<Class:0x1bb5698>:0x1bb5608> (NoMethodError)
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:35:in `gsub'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:35:in `do_process_handlebars'
        from C:/Sites/mysh/lib/mysh/user_input/handlebars.rb:26:in `eval_handlebars'
        from lib/mysh.rb:24:in `run'
        from lib/mysh.rb:57:in `<main>'
```

**Comment by PeterCamilleri** [comment]
Fixed in the add_variables branch.

**Comment by PeterCamilleri** [comment]
Fixed in version 0.3.0
